components/TimeLineChartCountyComparison.vue

```diff
@@ -177,25 +174,30 @@ export default {
       }
     },
     displayData() {
-      const bayAreaAverage = { color: '#2d2d2d', name: 'Bay Area Average' }
-      const countiesToDisplay = [
-        ...this.selectedCounties,
-        ...(this.overlays.average.selected ? [bayAreaAverage] : [])
-      ]
+      const countiesToDisplay = this.selectedCounties.concat(
+        this.overlays.average.selected ? ['totals'] : []
+      )
       const displayTiers = this.overlays.tiers?.selected
       if (countiesToDisplay.length) {
         const dataSets = []
         const data =
           this.chartDataType === 'casesperpeople'
             ? this.caseData
             : this.percentData
-        const labels = this.chartData[countiesToDisplay[0].name].graph.map(
-          d => {
-            return d.label
+        const chartData = this.chartData
+        const labels = countiesToDisplay.reduce((ls, county) => {
```

Comment:
Changing the way the list of labels are created as well as the how the length of the time slice array is calculated fixes issue#1090

Reply:
Okay!